src/pydistcheck/__init__.py

```diff
@@ -1,4 +1,4 @@
 # no one should be importing from this package
 __all__ = []  # type: ignore[var-annotated]
 
-__version__ = "0.8.0.99"
+__version__ = "0.8.1"
```

Comment:
```suggestion
__version__ = "0.8.0.99"
```

Please revert this. This change won't be immediately released.